packages/augur-core/source/contracts/Augur.sol

```diff
@@ -489,6 +498,27 @@ contract Augur is IAugur {
         return true;
     }
 
+    function logParticipationTokensTransferred(IUniverse _universe, address _from, address _to, uint256 _value, uint256 _fromBalance, uint256 _toBalance) public returns (bool) {
+        require(isKnownUniverse(_universe));
+        require(_universe.isContainerForDisputeWindow(IDisputeWindow(msg.sender)));
+        emit TokensTransferred(address(_universe), msg.sender, _from, _to, _value, TokenType.ParticipationToken, address(0), _fromBalance, _toBalance);
+        return true;
+    }
+
+    function logParticipationTokenBurned(IUniverse _universe, address _target, uint256 _amount, uint256 _totalSupply) public returns (bool) {
```

Comment:
Nit: the log functions for minting/burning are not named consistently. (Eg, some end in “TokenMinted” & some end in “TokensMinted.”) Recommend making them all use “Tokens” instead of “Token.”